en/sql/function/arithmetic_op.rst

```diff
@@ -233,27 +235,42 @@ The following table shows the total number of digits (*p*) and the number of dig
                                                1.524157875322756e+024
      
 .. code-block:: sql
+    csql> ;get oracle_compat_number_behavior
+    oracle_compat_number_behavior=n
 
     -- int / int returns int type without type conversion or rounding
     SELECT 100100/100000;
-    
+
 ::
 
       100100/100000
     ===============
                   1
-     
+
 .. code-block:: sql
+    csql> ;get oracle_compat_number_behavior
+    oracle_compat_number_behavior=n
 
     -- int / int returns int type without type conversion or rounding
     SELECT 100100/200200;
-    
+
 ::
 
       100100/200200
     ===============
                   0
-     
+
+.. code-block:: sql
+    csql> ;get oracle_compat_number_behavior
```

Comment:
How about add a statement to set the parameter to yes ?
csql> ;set oracle_compat_number_behavior=y

Reply:
this parameter is sever-parameter, so the setting by csql is not allowed.

Reply:
OK, I see.